client/src/components/auth/Register.js

```diff
@@ -93,4 +103,4 @@ const mapStateToProps = (state) => ({
   errors: state.errors
```

Comment:
This piece of code is missing: 
const mapStateToProps = state => ({
  auth: state.auth,
  errors: state.errors
});